DependencyInjection/Compiler/AddTreeBrowserAssetsPass.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\DoctrinePHPCRAdminBundle\DependencyInjection\Compiler;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Definition;
+use Symfony\Component\DependencyInjection\Reference;
+
+/**
+ * Adds javascripts and stylesheets required for the TreeSelectType.
+ *
+ * @author Wouter de Jong <wouter@wouterj.nl>
+ */
+class AddTreeBrowserAssetsPass implements CompilerPassInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function process(ContainerBuilder $container)
+    {
+        if (!$container->hasDefinition('sonata.admin.pool')) {
+            return;
+        }
+
+        $definition = $container->findDefinition('sonata.admin.pool');
+        $this->addAssetsToAdminPool($definition);
```

Comment:
Please remove the one-time variable : `$this->addAssetsToAdminPool($container->findDefinition('sonata.admin.pool'));`